src/eth/storage/permanent/rocks/types/transaction_mined.rs

```diff
@@ -3,44 +3,44 @@ use std::fmt::Debug;
 use super::block_number::BlockNumberRocksdb;
 use super::execution::ExecutionRocksdb;
 use super::hash::HashRocksdb;
-use super::index::IndexRocksdb;
-use super::log_mined::LogMinedRockdb;
+use super::log_mined::LogMinedRocksdb;
 use super::transaction_input::TransactionInputRocksdb;
+use crate::eth::primitives::Index;
 use crate::eth::primitives::LogMined;
 use crate::eth::primitives::TransactionMined;
 
 #[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize, fake::Dummy)]
 pub struct TransactionMinedRocksdb {
     pub input: TransactionInputRocksdb,
     pub execution: ExecutionRocksdb,
-    pub logs: Vec<LogMinedRockdb>,
-    pub transaction_index: IndexRocksdb,
```

Comment:
Although there is some redundancy keeping the transaction index when theoretically the transactions array inside the block should be ordered, I feel like we should keep the index in the transaction still, and use that instead of enumerating the transactions array.
This is more of a sanity thing, I don't recall us having some strong mechanism that guarantees that we order the transactions array by the transaction index, so I don't feel we should rely on the ordering.